src/components/ProductDetail/CommentList.jsx

```diff
@@ -0,0 +1,67 @@
+import { useState } from "react";
+
+import "./CommentList.css";
+import inquiryEmptyImage from "../../assets/images/Img_inquiry_empty.png";
+
+import Comment from "./Comment";
+
+function CommentList({ comments }) {
+  const [isInputFill, setIsInputFill] = useState(false);
+
+  const handleValueChange = (e) => {
+    setIsInputFill(e.target.value !== "");
+  };
+
+  const buttonClassName = isInputFill
+    ? "add-inquiry-button active"
+    : "add-inquiry-button";
+
+  const placeholder =
+    "개인정보를 공유 및 요청하거나 명예 훼손, 무단 광고, 불법 정보 유포 시 모니터링 후 삭제될 수 있으며, " +
+    "이에 대한 민형사상 책임은 게시자에게 있습니다.";
+
+  return (
+    <>
+      <div className="add-inquiry-wrapper">
+        <label className="inquiry-label" htmlFor="inquiry">
+          문의하기
+        </label>
+        <textarea
+          className="inquiry-content"
+          id="inquiry"
+          placeholder={placeholder}
+          onChange={handleValueChange}
+        ></textarea>
+        <button className={buttonClassName} disabled={!isInputFill}>
+          등록
+        </button>
+      </div>
+
+      <div className="inquiry-wrapper">
+        {comments.length !== 0 ? (
```

Comment:
p4;
요런 경우에 comment.length가 0인지 아닌지에 따라 boolean으로 형변환을 하여 아래와 같이 써도 괜찮을 것 같아요!
```suggestion
        {!!comments.length ? (
```